src/main/java/Rolex.java

```diff
@@ -1,89 +1,115 @@
             }
-
             else if(UserInput.equalsIgnoreCase("list")){
-                printLines();
-                int indexNum = 1;
-                System.out.println("Here are the tasks in your list:");
-                for(int i = 0; i<taskCount; i++){
-                    System.out.println(indexNum + ". " + task[i]);
-                    indexNum++;
-                }
-                printLines();
+                InputIsList();
             }
-
             else if(UserInput.startsWith("mark")){
-                int index = Integer.parseInt(UserInput.substring(5));
-                task[index-1].MarkTask();
+                InputIsMark(UserInput);
             }
-
             else if(UserInput.startsWith("unmark")){
-                int index = Integer.parseInt(UserInput.substring(7));
-                task[index-1].unMarkTask();
+                InputIsUnmark(UserInput);
             }
-
             else if(UserInput.startsWith("todo")){
-                printLines();
-                String todoName = UserInput.substring(5);
-                task[taskCount] = new Todo(todoName);
-                taskCount++;
-                System.out.println("Got it. I've added this task:");
-                System.out.println(task[taskCount-1]);
-                System.out.println("\nNow you have " + taskCount + " tasks in the list.");
-                printLines();
+                InputIsTodo(UserInput);
             }
-
             else if(UserInput.startsWith("deadline")){
-                printLines();
-                int indexOfBy =   UserInput.indexOf("/by");
-                String deadlineName = UserInput.substring(9,indexOfBy-1);
-                String by = UserInput.substring(indexOfBy+3);
-                task[taskCount] = new Deadline(deadlineName, by);
-                taskCount++;
-                System.out.println("Got it. I've added this task:");
-                System.out.println(task[taskCount-1]);
-                System.out.println("\nNow you have " + taskCount + " tasks in the list.");
-                printLines();
-
+                InputIsDeadline(UserInput);
             }
-
             else if(UserInput.startsWith("event")){
-                printLines();
-                int indexOfFrom = UserInput.indexOf("/from");
-                int indexOfTo = UserInput.indexOf("/to");
-                String eventName = UserInput.substring(6,indexOfFrom-1);
-                String startTime = UserInput.substring(indexOfFrom+6,indexOfTo-1);
-                String endTime = UserInput.substring(indexOfTo+4);
-                task[taskCount] = new Event(eventName, startTime, endTime);
-                taskCount++;
-                System.out.println("Got it. I've added this task:");
-                System.out.println(task[taskCount-1]);
-                System.out.println("\nNow you have " + taskCount + " tasks in the list.");
-                printLines();
+                InputIsEvent(UserInput);
```

Comment:
'switch' is more better than if-else statement in this case since there are various cases you want to separate.

Reply:
Actually, I have changed the coding standard and quality after doing my level-4. In that, most of the lines u see in red are inside separate functions. They look short and concise now. Thanks for your feedback!